modules/authentication/proc-enabling-authentication-with-microsoft-azure.adoc

```diff
@@ -33,45 +38,41 @@ To grant admin consent, a directory admin must go to the link:https://learn.micr
+`AUTH_AZURE_TENANT_ID`:: Enter your *Tenant ID* on Microsoft Azure.
 
-. Set up the Microsoft Azure authentication provider in your {product-short} custom configuration.
-+
-Edit your custom {product-short} config map, such as `app-config-rhdh`.
-+
-In the `app-config-rhdh.yaml` content, add the `microsoft` provider configuration under the root `auth` configuration, and enable the `microsoft` provider for sign-in:
+. Set up the Microsoft Azure authentication provider in your {product-short} custom configuration, such as `app-config-rhdh`:
 +
 .`app-config-rhdh.yaml` fragment
 [source,yaml,subs="+quotes,+attributes"]
 ----
 auth:
-  environment: production
+  environment: production # <1>
   providers:
     microsoft:
       production:
         clientId: ${AUTH_AZURE_CLIENT_ID}
         clientSecret: ${AUTH_AZURE_CLIENT_SECRET}
         tenantId: ${AUTH_AZURE_TENANT_ID}
-        # domainHint: ${AUTH_AZURE_TENANT_ID} # <1>
-        # additionalScopes: # <2>
-          # - Mail.Send
-signInPage: microsoft # <3>
+        # domainHint: ${AUTH_AZURE_TENANT_ID} # <2>
+        # additionalScopes: # <3>
+        #   - Mail.Send
+signInPage: microsoft # <4>
+dangerouslyAllowSignInWithoutUserInCatalog: true # <5>
 ----
 +
-<1>  Optional for single-tenant applications. You can reduce login friction for users with accounts in multiple tenants by automatically filtering out accounts from other tenants. If you want to use this parameter for a single-tenant application, uncomment and enter the tenant ID. If your application registration is multi-tenant, leave this parameter blank. For more information, see link:https://learn.microsoft.com/en-us/azure/active-directory/manage-apps/home-realm-discovery-policy[Home Realm Discovery].
-<2> Optional for additional scopes. To add scopes for the application registration, uncomment and enter the list of scopes that you want to add.
+<1> Marking the environment as production hides the **Guest** login in the {product-short} home page.
+<2> Optional for single-tenant applications. You can reduce login friction for users with accounts in multiple tenants by automatically filtering out accounts from other tenants. If you want to use this parameter for a single-tenant application, uncomment and enter the tenant ID. If your application registration is multi-tenant, leave this parameter blank. For more information, see link:https://learn.microsoft.com/en-us/azure/active-directory/manage-apps/home-realm-discovery-policy[Home Realm Discovery].
+<3> Optional for additional scopes. To add scopes for the application registration, uncomment and enter the list of scopes that you want to add.
 The default and mandatory value is `['user.read']`.
-<3> To enable the Microsoft Azure provider as default sign-in provider.
+<4> To enable the Microsoft Azure provider as default sign-in provider.
+<5> To enable authentication without provisioning the software catalog.
```

Comment:
Could be useful to specify the provisioning of **users** specifically.

Reply:
Today I understand better. Of course:

```suggestion
<5> To enable authentication without provisioning users to the software catalog.
```